pkg/webhooks/pod/mutating.go

```diff
@@ -28,7 +40,7 @@ func (m *MutatingAdmission) Default(ctx context.Context, obj runtime.Object) err
 		return fmt.Errorf("expected a Pod but got a %T", obj)
 	}
 
-	klog.Infof("Into Pod injection %s/%s", pod.Namespace, pod.Name)
+	klog.Infof("mutating started for pod %s/%s", pod.Namespace, pod.Name)
```

Comment:
A code standard, all logs start with Capital letters, and please check if a log level should be added and disabled the msg by default. same suggestion to all newly added logs